apps/asap-server/test/controllers/research-outputs.test.ts

```diff
@@ -454,15 +480,15 @@ describe('ResearchOutputs controller', () => {
       test('Should pass the filter parameter as a squidex filter', async () => {
```

Comment:
i guess we should probably test searching for Proposals and Grant Documents separately - i would expect the first one to use just the Proposal filter and the other one - both?